src/install.sh

```diff
@@ -576,30 +574,26 @@ function create_k8s_custom_config() {
     local config_s3_path="s3://${ARMORY_CONF_STORE_BUCKET}/front50/config_v2/config.json"
     local config_gs_path="gs://${ARMORY_CONF_STORE_BUCKET}/front50/config_v2/config.json"
     if [[ "${CONFIG_STORE}" == "S3" ]]; then
-      aws --profile "${AWS_PROFILE}" --region us-east-1 s3 ls "${config_s3_path}" > /dev/null 2>&1
-      result=$?
-      if [[ $result -eq 0 ]]; then
+      if aws --profile "${AWS_PROFILE}" --region us-east-1 s3 ls "${config_s3_path}" > /dev/null 2>&1; then
         get_var "config.json already exists, would you like to overwrite it? [y/n]: " OVERWRITE_CONFIG
       fi
-      if [[ "${OVERWRITE_CONFIG}" == "y" ]] ; then
+      if [[ "${OVERWRITE_CONFIG}" != "n" ]] ; then
         echo "Overwriting config.json..."
         aws --profile "${AWS_PROFILE}" --region us-east-1 s3 cp \
           "${config_file}" \
           "${config_s3_path}"
       else
-        echo "Using existing custom-credentials.json..."
+        echo "Using existing config.json..."
```

Comment:
i.... think i'm confused with what this function is doing, and what `echo` is doing...